crates/humanode-peer/src/chain_spec.rs

```diff
@@ -1,26 +1,49 @@
 //! Provides the [`ChainSpec`] portion of the config.
 
 use crypto_utils::{authority_keys_from_seed, get_account_id_from_seed};
-use frame_support::BoundedVec;
+use frame_support::{BasicExternalities, BoundedVec};
 use hex_literal::hex;
 use humanode_runtime::{
     opaque::SessionKeys, robonode, token_claims::types::ClaimInfo, AccountId, BabeConfig, Balance,
     BalancesConfig, BioauthConfig, BootnodesConfig, ChainPropertiesConfig, EVMConfig,
-    EthereumAddress, EthereumChainIdConfig, EthereumConfig, EvmAccountId, GenesisConfig,
-    GrandpaConfig, ImOnlineConfig, SessionConfig, Signature, SudoConfig, SystemConfig,
-    TokenClaimsConfig, WASM_BINARY,
+    EnableDevBlockImportSeal, EthereumAddress, EthereumChainIdConfig, EthereumConfig, EvmAccountId,
+    GenesisConfig, GrandpaConfig, ImOnlineConfig, SessionConfig, Signature, SudoConfig,
+    SystemConfig, TokenClaimsConfig, WASM_BINARY,
 };
 use pallet_im_online::sr25519::AuthorityId as ImOnlineId;
 use sc_chain_spec_derive::{ChainSpecExtension, ChainSpecGroup};
 use sc_service::ChainType;
 use serde::{Deserialize, Serialize};
 use sp_consensus_babe::AuthorityId as BabeId;
 use sp_finality_grandpa::AuthorityId as GrandpaId;
-use sp_runtime::{app_crypto::sr25519, traits::Verify};
+use sp_runtime::{app_crypto::sr25519, traits::Verify, Storage};
 
 /// The concrete chain spec type we're using for the humanode network.
 pub type ChainSpec = sc_service::GenericChainSpec<humanode_runtime::GenesisConfig, Extensions>;
 
+/// Specialized `ChainSpec` for development.
+pub type DevChainSpec = sc_service::GenericChainSpec<DevGenesisConfig, Extensions>;
+
+/// Extension for the dev genesis config to support a custom changes to the genesis state.
+#[derive(Serialize, Deserialize)]
+pub struct DevGenesisConfig {
+    /// Genesis config.
+    genesis_config: GenesisConfig,
+    /// The flag that if enable block import manual seal mode.
+    enable_block_import_manual_seal: Option<bool>,
```

Comment:
```suggestion
    enable_block_import_manual_seal: bool,
```
We can just use a boolean here and set to true if it has to be true.

This can also be used to embed arbitrary robonode key into the runtime, i.e. for benchmarks.

But then again - why not create a full-blown pallet for this? If we need a storage and and genesis config - why bother and do this magic?